tiledb/api/src/array/schema/mod.rs

```diff
@@ -984,7 +1009,12 @@ mod tests {
         .unwrap();
 
         let s: Schema = b.build().unwrap();
-        assert_eq!(0, s.version().unwrap());
+
+        let schema_version = s.version().unwrap();
+
+        // it's a little awkward to use a constant here but this does
+        // not appear to be tied to the library version
```

Comment:
Correct. The schema version is actually the underlying data format which is incremented anytime we change how something is stored on disk. You can find the mapping of storage format versions to library versions here:

https://github.com/TileDB-Inc/TileDB/blob/dev/format_spec/FORMAT_SPEC.md#history